### PR TITLE
utils_net: enable get_network_cfg_file() and create_network_script() for host

### PR DESCRIPTION
**Pull request by balamuruhans**
Enable APIs to provide distro specific network config file and create network
script in host and guest

Signed-off-by: Balamuruhan S <bala24@linux.vnet.ibm.com>

### CODE REVIEW

File: virttest/utils_net.py [hunk 1]
```diff
@@ -1609,21 +1613,24 @@ def get_vm_network_cfg_file(vm, iface_name):
     return iface_cfg_file
 
 
-def create_vm_network_script(vm, iface_name, mac_addr, boot_proto, net_mask,
-                             ip_addr=None, **dargs):
+def create_network_script(iface_name, mac_addr, boot_proto, net_mask,
+                          vm=None, ip_addr=None, **dargs):
     """
-    Form network script with its respective network param for vm instance
+    Form network script with its respective network param for vm or Host.
```
Comment (xutian) [inline virttest/utils_net.py]: Better to doc what should include in  dargs dict.

Comment (xutian) [inline virttest/utils_net.py]: View from function name,  this function use to generate configuration file to network service, but  checked it implementation, it looks not and it just return  param list. And the function is not complete.

Comment (balamuruhans) [inline virttest/utils_net.py]: @xutian, sure I will work it out.

**Comment by balamuruhans** [comment]
@xutian, I have made the changes suggested by you.

**Comment by balamuruhans** [comment]
Thanks @xutian , had made API to generate configuration file instead of returning the params list.
Merging now.

**Pull request merged**